### PR TITLE
refactor(lib/eoa): improve and refactor eoa funding

### PR DESCRIPTION
**Pull request by corverroos**
Refactor and improve eoa package and funding.

task: none

### CODE REVIEW

**Review by pavelnikolov — APPROVED** [review]

File: e2e/app/eoa/static.go [hunk 1]
```diff
@@ -1,285 +1,45 @@
...
-
-	minBalanceMedium    = ether1
-	targetBalanceMedium = ether5
-
-	minBalanceLarge    = ether10
-	targetBalanceLarge = ether100
+const (
+	// fbFunder is the address of the fireblocks "funder" account.
+	fbFunder  = "0xf63316AA39fEc9D2109AB0D9c7B1eE3a6F60AEA4"
+	fbDev     = "0xf63316AA39fEc9D2109AB0D9c7B1eE3a6F60AEA4"
```
Comment (ttarsi) [inline e2e/app/eoa/static.go:11]: should this be the fb dev address?

**Review by kevinhalliday — APPROVED** [review]

File: e2e/app/eoa/static.go [hunk 2]
```diff
@@ -1,285 +1,45 @@
...
+		remote("0x0CdCc644158b7D03f40197f55454dc7a11Bd92c1", RoleDeployer),
+		secret("0x01654f55E4F5E2f2ff8080702676F1984CBf7d8a", RoleRelayer),
+		secret("0x12Dc870b3F5b7f810c3d1e489e32a64d4E25AaCA", RoleMonitor),
+		fbDevAcc(),
+	),
+	netconf.Mainnet: flatten(
+		dummy(RoleProxyAdminOwner, RolePortalAdmin, RoleAVSAdmin, RoleCreate3Deployer, RoleDeployer),
+		secret("0x07804D7B8be635c0C68Cdf3E946114221B12f4F7", RoleRelayer),
+		secret("0x07082fcbFA5F5AC9FBc03A48B7f6391441DB8332", RoleMonitor),
+		fbDevAcc(),
```
Comment (ttarsi) [inline e2e/app/eoa/static.go:43]: don't think we should include this here as it's part of our fireblocks testnet workspace, not the mainnet one

**Review by ttarsi — APPROVED** [review]
approving with 2 nits

File: e2e/app/eoa/static.go (hunk 2 above)
Comment (corverroos) [reply e2e/app/eoa/static.go]: cool, replace with `dummy` for now.

File: e2e/app/eoa/static.go (hunk 1 above)
Comment (corverroos) [reply e2e/app/eoa/static.go]: thanks

**Pull request merged**